src/idc/zephyr_idc.c

```diff
@@ -110,8 +110,11 @@ int idc_send_msg(struct idc_msg *msg, uint32_t mode)
 
 	idc_send_memcpy_err = memcpy_s(msg_cp, sizeof(*msg_cp), msg, sizeof(*msg));
 	assert(!idc_send_memcpy_err);
-	/* TODO: verify .priority and .deadline */
-	work->priority = K_HIGHEST_THREAD_PRIO + 1;
+	/*
+	 * Same priority as the IPC thread which is an EDF task and under Zephyr
+	 * its priority is set in scheduler_init_edf() to 1
+	 */
+	work->priority = 1;
```

Comment:
OK, just pls use a meaningful macro - same here and in scheduler_init_edf

Reply:
@marcinszkudlinski done